packages/stranger/src/things/component.js

```diff
@@ -17,9 +17,10 @@ const comp = (styles = {}) => (Component = 'div') => {
       ? merge({}, stylesObj, existingStyles)
       : stylesObj
     const className = addRule(mergedStyles)
+    const newProps = { ...rest, styles: mergedStyles }
     const propsToPass = Component.isStrange
-      ? { ...rest, styles: mergedStyles }
-      : pickBy(rest, validAttr(Component.propTypes))
+      ? newProps
```

Comment:
This feels like a bugfix on Stranger's comp component. Split it out into another branch and PR it separately so we can have a changelog in the future? (Sorry, am all about doccing right now)